### DISCUSSION TITLE
How to create proper transaction using "New Web3.js - v2" and send it using phantom wallet "window.phantom.solana" ?

### DISCUSSION THREAD

**Discussion by pooriagg** [phantom/docs#361] [Help & Support] [answered]
### Summary

How to create proper transaction using "New Web3.js - v2" and send it using phantom wallet "window.phantom.solana" ?

### Steps to Reproduce

How to create proper transaction using "New Web3.js - v2" and send it using phantom wallet "window.phantom.solana" ?

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hello @pooriagg,

From my understanding the same provider calls detailed in our [docs](https://docs.phantom.com), should work. Please let me know if you have a specific issue or error.
